src/content/docs/dev/send-tokens/interchain-tokens/integrate-custom-token.mdx

```diff
@@ -39,42 +39,79 @@ contract MyInterchainToken is ERC20, ERC20Burnable {
 }
 ```
 
-This token is a simple implementation of an ERC-20 token but includes the critical functionality needed to integrate with ITS: the ability to `mint()` and `burn()` tokens. ITS interacts with these functions when bridging tokens between chains.
+This token is a simple implementation of an ERC-20 token that includes the critical functions needed to integrate with ITS: the ability to `mint()` and `burn()` tokens. ITS calls these functions when bridging tokens between chains.
 
-You also have the option to inherit from the [InterchainTokenStandard](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interchain-token/InterchainTokenStandard.sol#L12) so that your token has cross-chain functionality built-in, such as the `interchainTransfer()` function.
+Alternatively, you can inherit from the [InterchainTokenStandard](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interchain-token/InterchainTokenStandard.sol#L12) so that your token has cross-chain functionality built in, such as the `interchainTransfer()` function.
 
-## Integrate the Custom Token with ITS by Deploying a Token Manager
+## Integrate Custom Token with ITS
 
-Use the `deployTokenManager` function to create a Token Manager for your token:
+After deploying your custom tokens on your preferred chains, the next step is to integrate your token with ITS.
+
+### Register Token Metadata with the ITS Contract
+
+Register your token metadata using the [`registerTokenMetadata`](https://github.com/axelarnetwork/interchain-token-service/blob/94677f1072da127a0ee7ff5ab6784dcfe11789e8/contracts/InterchainTokenService.sol#L274) function on the ITS contract. For example, if you are integrating your custom token on chains A, B, and C, you should register token metadata on each chain separately.
+
+```solidity
+function registerTokenMetadata(
+    address tokenAddress,  // your token address
+    uint256 gasValue       // gas value
+) external payable {};
+```
+
+This function registers metadata for a token on the [ITS Hub](#). This metadata is used for scaling linked tokens.
+
+> **Note:** Token metadata must be registered before `linkToken` can be called for the corresponding token.
```

Comment:
we should explain what `linkToken` is before mentioning it.

Reply:
or perhaps link to where it is explained

Reply:
Added a follow-up explanation